common/src/main/java/com/wynntils/wynn/event/TotemActivatedEvent.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wynn.event;
+
+import com.wynntils.mc.objects.Location;
+
+public class TotemActivatedEvent extends TotemEvent {
+    private final int time;
+    private final Location location;
+
+    public TotemActivatedEvent(int totemNumber, int time, Location location) {
```

Comment:
By tradition, when we have abstract events, we have put the concrete subclasses as static inner classes. I think it is kind of a nice pattern to follow, so you'd get a `TotemEvent.Activated` instead, etc.